### PR TITLE
Blog: The future of YAML

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Blog post about "The future of YAML".

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by balloob — DISMISSED** [review]

File: source/_posts/2020-04-14-the-future-of-yaml.markdown [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+This halts our project goals, slows down innovation and induces the risk of
+losing contributors and maintainers. In the end, leads to a greater loss for
+everybody.
+
+## The future of YAML
+
+So, with all of the above set context: What is the future of YAML?
+
+As of today, an ADR (Architectural Decision Record) has been approved that
```
Comment (balloob) [inline source/_posts/2020-04-14-the-future-of-yaml.markdown]: Can we link the ADR

File: source/_posts/2020-04-14-the-future-of-yaml.markdown [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+states:
+
+- Any new integration that communicates with devices and/or services, must use
+  configuration via the UI. Configuration via YAML is only allowed in very rare
+  cases, which will be determined on a case by case basis.
```
Comment (dshokouhi) [inline source/_posts/2020-04-14-the-future-of-yaml.markdown:150]: Should we give an example of a rare case?

Comment (frenck) [reply source/_posts/2020-04-14-the-future-of-yaml.markdown:150]: There is no strict ruling on that. Giving an example, loosen the ADR, which I think we should not do.

Furthermore, any case stated at that point, will not be forever.

File: source/_posts/2020-04-14-the-future-of-yaml.markdown (hunk 1 above)
Comment (frenck) [reply source/_posts/2020-04-14-the-future-of-yaml.markdown]: Done 👍

**Comment by balloob** [comment]
🎉

**Pull request merged**